### ISSUE TITLE
Documentation: Backups handbook.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#3958]
While working on #3957, I've noticed that we are pointing to https://www.drupal.org/node/22281 for instructions on backing up a site. I have changed that link in my PR to point to https://backdropcms.org/upgrade instead, which I believe suffices.

Question is: do we need all the detailed instructions from that d.org article on how to back up a site via the command line, backup strategy etc.? If yes, then we should copy those over to b.org, and update them to match Backdrop specificities (config backup etc.).

**Comment by klonos** [comment]
...noting that in https://backdropcms.org/upgrade we do not mention backing up the files directory or the entire codebase - only the config directory.

**Comment by klonos** [comment]
...also noting that for some reason https://backdropcms.org/upgrade and https://backdropcms.org/upgrade-from-drupal have been created as an "About page" type, instead of "Documentation" pages.

**Comment by klonos** [comment]
...https://backdropcms.org/admin/structure/types/manage/docs#edit-menu had only "User Guide" as available menus, which may explain why the original author of these pages chose to go with "About page" instead of "Documentation". I have added the "Developer Documentation" and "Handbook" menus to it.

I will start moving them to the proper content type, and replace the old articles with the new ones. I wish we had https://www.drupal.org/project/node_convert ported.

**Comment by klonos** [comment]
...I have moved all content into "Documentation" pieces of content (retained redirects, menu settings, book outline etc.).

Now, should we add a "Backing up a site" section to the dev docs?

**Comment by olafgrabienski** [comment]
> Now, should we add a "Backing up a site" section to the dev docs?

Yes, I think it's more useful than a link to https://backdropcms.org/upgrade. Copy https://www.drupal.org/node/22281 and change it to match Backdrop specificities sounds like a good start!

**Comment by klonos** [comment]
I have started a draft at https://backdropcms.org/backup

...my view is that the respective "General backup best practices" section is not very professionally written, and offers very little backup-specific value. At the very least, I would like us to demote its prominence, by not having it be the first think people read in our backups guide. It can remain a subsection (separate page perhaps), at the end of the main documentation section about backups.

So, I would like to propose the following structure:

User Guide
- [About Backdrop CMS](https://backdropcms.org/about)
- [Developers notes](https://backdropcms.org/developers)
  - System Requirements
  - Installation Instructions
  - Updates and Upgrades
  - Upgrading from Drupal 7
  - **[Backing up a Backdrop site](https://backdropcms.org/backup)**
    - Backing up your site using the command line
    - Backing up your site using the admin UI
    - Restoring a backup
    - **General backup best practices**
  - How to use the core issue queue
  - Submitting Core Pull Requests

**Comment by olafgrabienski** [comment]
> So, I would like to propose the following structure:
>
> (...)
>   - **[Backing up a Backdrop site](https://backdropcms.org/backup)**
>     - Backing up your site using the command line
>     - Backing up your site using the admin UI
>     - Restoring a backup
>     - **General backup best practices**

Looks overall good to me, but I'd suggest to drop "Backdrop" in the heading. Alternatives: "Backing up a site" or "Backing up your site".

**Comment by klonos** [comment]
> ...I'd suggest to drop "Backdrop" in the heading.

The instructions we'll be providing will be specific to Backdrop though. See how the previous item is "Upgrading from Drupal 7"?

**Comment by olafgrabienski** [comment]
> See how the previous item is "Upgrading from Drupal 7"?

Yes, I saw that, but it is the only Drupal item. All the other items are about Backdrop, and consequently there is no need to mention "Backdrop". That said, I understand, that having "Drupal" directly above the "Backing up" item, we may want to make sure that we're not talking about Drupal anymore.

**Comment by klonos** [comment]
> ...we may want to make sure that we're not talking about Drupal anymore.

Nor that we may be talking about generic website backups. Backdrop is "special" in its own way, in that we need to also backup configuration files besides the database; and our config files work differently than the D8 config files. So yeah, not D7, not D8, and not generic instructions on how to back up a generic/plain "site"; this is specific to Backdrop site backups 😉
